server/chat-plugins/info.js

```diff
@@ -2397,55 +2397,39 @@ const pages = {
 		let buf = "";
 		this.extractRoom();
 		if (!user.named) return Rooms.RETRY_AFTER_LOGIN;
-		buf += `<div class="pad"><h2>List of active punishments:</h2>`;
+		if (!this.room.chatRoomData) return;
```

Comment:
So the command refuses a room with a hyphen but this line doesn't. Also this should still return the html you removed earlier shouldn't it.

Reply:
> the command refuses a room with a hyphen but this line doesn't.

I added that line because for some reason in the page it catches battle rooms while in the commands it still continued and I didn't care enough to investigate the reasoning 
> this should still return the html you removed earlier shouldn't it.

I don't see a reason to how 'List of active punishments:' if it's an invalid room.